builtins.isph

```diff
@@ -0,0 +1,740 @@
+// -*- mode: c++ -*-
+/*
+  Copyright (c) 2024, Intel Corporation
+
+  SPDX-License-Identifier: BSD-3-Clause
+*/
+
+/** @file builtins.isph
+
+    @brief This file contains the declarations of the built-in functions that
+    are defined in builtins*.ll files but used in the ISPC code of standard
+    library. These functions should not be used outside of the standard library
+    code.
+*/
+
+// Macros for attributes
+#define NOESCAPE __attribute__((noescape))
+#define ADDRSPACE(N) __attribute__((address_space(N)))
+// #define READONLY __attribute__((memory("read")))
+#define READONLY
+// #define READNONE __attribute__((memory("none")))
+#define READNONE
+// #define NOALIAS __attribute__((noalias))
+#define NOALIAS
+
+extern "C" varying int32 __padds_vi32(varying int32, varying int32);
```

Comment:
I think all builtin functions should be `inline`.
Have you figure out the effect of lost attributes like `unwind`?

Reply:
> I think all builtin functions should be `inline`.

It is not so at the moment. 

Moreover, I think that forcing inlining (always and unconditionally) can actually be harmful. LLVM should decide better whether the inlining of the particular function is profitable. Although this consideration is unrelated to the current change.

> Have you figure out the effect of lost attributes like `unwind`?

It is not lost. It is populated on all functions.

Reply:
> this consideration is unrelated to the current change.

Agree.

> It is not lost. It is populated on all functions.

I'm referring to the functions that were moved from `utils.m4` to `core.isph`, such as the `pseudo*`... functions. I’ve noticed that these functions are marked as nounwind in the generated LLVM IR. However, I'm unable to locate where this marking is applied—whether it's explicitly set somewhere or if LLVM is automatically applying it based on its own logic.